Add Quick.JS to jsdelivr

MK2018 opened the pull request:
This is a pull request with both the info.ini & the update.json, as per the contribution guidlines, for adding Quick.JS to jsdelivr

jimaek commented on files/quickjs/update.json:
```diff
@@ -0,0 +1,9 @@
+  "name": "quickjs",
+  "repo": "mk2018/quickjs",
+  "files": {
+    "basePath": "builds/0.1a/",
```
This format wont work, it will break the minute you release a new version.
Also there is no documentation or website.

MK2018 commented:
I changed the file structure so that it wouldn't break, and added some basic documentation. As of right now, Quick.JS is still in development, so more documentation is forthcoming, but it'll be done very soon. Hopefully this helps.

jimaek commented:
I recommend waiting until the code is more stable and production ready before submitting.
Leave a comment here to review it again.

MK2018 commented:
Thanks, I'll do a new pull request when it's ready.
